SourceryRuntime/Sources/TemplateContext.swift

```diff
@@ -230,4 +230,7 @@ extension ProcessInfo {
         }
     }
 
+    public override func responds(to aSelector: Selector!) -> Bool {
+        return true
```

Comment:
Now Stencil is smarter and on `NSObject`s calls this method before using KVC, so we have to override it to be able to access types by their names.